compositions/upbound-aws-provider/lambda/container.yaml

```
@@ -46,6 +95,10 @@ spec:
           deletionPolicy: Delete
           forProvider:
             packageType: Image
+            role: "tbd"
+            environment:
+              - variables:
+                  hardcoded: value
```

```suggestion
            environment:
              - variables: {}
```
can make this shorter like this